recipe/meta.yaml

```
@@ -44,6 +44,7 @@ requirements:
     - python
     - libudev  # [linux and cdt_name!='cos6']
     - libusb
+    - cuda-version {{ cuda_compiler_version }}  # [(cuda_compiler_version or "None") != "None"]
```

It looks like CMake can't find `cudart`. So let's add it as a dependency and see if that helps

We should double check whether it is linking to `cudart` or not. If not, we may want to add this to `ignore_run_exports_from` above

```suggestion
    - cuda-version {{ cuda_compiler_version }}  # [(cuda_compiler_version or "None") != "None"]
    - cuda-cudart-dev                           # [(cuda_compiler_version or "None") != "None"]
```

---

Ah, I missed this comment and added it like this https://github.com/conda-forge/librealsense-feedstock/pull/56/commits/dc6ed151afa4b2f042ccc44aa32a8277dde3f1e5 (I added it only for cuda==12 builds).